commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/TestData.java

```diff
@@ -100,15 +122,26 @@ static Stream<Arguments> testCombine() {
         );
     }
 
-    static Stream<Arguments> testCombineNonFinite() {
+    /**
+     * Function which supplies data with non-finite values to test the <code>combine()</code> method.
```

Comment:
supplies a test data for a statistic in as a pair of double[] arrays. Each case will contain at least 1 non-finite value.